changelogs/unreleased/7253-remove-use-generic.yml

```diff
@@ -0,0 +1,5 @@
+description: Drop `use_generic` from `Resource.deserialize` method
+issue-nr: 7243
+change-type: patch
```

Comment:
I think merge tool will complain if we make this a patch but we don't merge it in branches that allow patches ie iso 6 and iso7. I think this has to be a major ?

Reply:
I don't think so but I might be mistaken. I think it only complains the other way around. @sanderr?

Reply:
It does complain under certain circumstances (not these). If the branches are still aligned, a patch is not allowed to diverge them. But the branches are already diverged so then anything goes.